wcivf/apps/elections/templates/elections/includes/_single_ballot.html

```diff
@@ -28,6 +28,9 @@ <h2>{{ postelection.friendly_name }}</h2>
         {% else %}
           {{ postelection.election.in_past|yesno:"was,will be" }} held
           <strong>{{ postelection.election.election_date|naturalday:"\o\n l j F Y" }}</strong>.
+          {% if postelection.next_ballot %}
+            <a href="{{ postelection.next_ballot.get_absolute_url }} ">Click here to view the upcoming election for this post.</a>
```

Comment:
Couple of points about this:

1. using "click here" is [generally not a good idea for links](https://webaccess.berkeley.edu/ask-pecan/click-here) - I know you've included the descriptive text after "click here", but I don't think that adds a lot. 

2. Related: it might be nicer state that we know an election is happening slightly differently, like "{{ place name }} has an election on the {{ date }}" - could check in with @pmk01 about what he thinks on this.

3. Can we use the same logic to get the nice area name, rather than "post"? Now we have it, I'm keen to use it everywhere :)

Sorry for the bike-shed about a single line!

Reply:
I agree with Sym! 

perhaps:
The next election for {{position}} is due to take place on {{date}}.

If we wanted to be really clever, we could link to the previous election as well? I don't know if that's useful for current elections, but may be for past ones. Happy to have more of a think and chat about this.

Relatedly, I really like the way Wikipedia election infoboxes provide links to the previous election with a little arrow and year:

![Screenshot 2021-02-15 122542](https://user-images.githubusercontent.com/61510609/107946465-f618fc00-6f88-11eb-8b3e-15ed00a7766d.png)

Reply:
I've moved the link to a new card that is only displayed on the `post_view.html`:

<img width="1552" alt="Screenshot 2021-02-15 at 16 35 11" src="https://user-images.githubusercontent.com/15347726/107972878-638a5400-6fac-11eb-89ea-f5dee6dc9243.png">

This will only display if the election is not the 'current' election and if there is a future election to display. But not sure if you think that it is overkill?

@pmk01 from a technical point, we can link to the previous election. But I'm not sure how/where the arrow example from Wikipedia would best work, so yes lets have a chat about it if its something you'd like to get working

Reply:
In fact now I am starting to wonder if I haven't potentially misinterpreted the original issue entirely. I have been thinking about it in terms of wanting to display a link to the next election that is taking place. So a past election would always link to the next one taking place (if it exists). But should it be the next election chronologically? So 2015 would link to 2017, then 2017 to 2019 etc?

Reply:
Oh, that is a good question. I like the text in your example and would be very happy to deploy that now. I suppose we could show a list of previous elections, though at some point that might get a bit busy... @symroe do you know which seat has had the most elections since we started tracking by-elections in 2016?

Reply:
I am actually very happy with either 'most recent' or 'next election'. Maybe we could ask the volunteers. 

It's not a bad thing if we don't make a complete decision right now, as after May I want to have a conversation about how we've become an archive and how we manage information across the board anyway.

Reply:
Perhaps that is good compromise then - we go with the above for now, where all past elections link to the upcoming election. As that would seem the most useful feature for right now anyway. And then after May we could revisit this to decide if we want to implement wider changes, discussing with the volunteers.